src/credentials/examples/DeviceAttestationVerifierExample.cpp

```diff
@@ -0,0 +1,221 @@
+              { 0xE7, 0xEF, 0xEA, 0xC3, 0x33, 0x5C, 0x5F, 0xD0, 0xC3, 0xE6,
+                0x34, 0x52, 0x9F, 0x16, 0x70, 0x46, 0xC4, 0xBC, 0xA5, 0x5C } },
+        };
+
+    size_t paaLookupTableIdx;
+    for (paaLookupTableIdx = 0; paaLookupTableIdx < ArraySize(sPAALookupTable); ++paaLookupTableIdx)
+    {
+        if (skid.data_equal(ByteSpan(sPAALookupTable[paaLookupTableIdx].mSKID)))
+        {
+            break;
+        }
+    }
+
+    VerifyOrReturnError(paaLookupTableIdx < ArraySize(sPAALookupTable), CHIP_ERROR_INVALID_ARGUMENT);
+
+    return CopySpanToMutableSpan(ByteSpan{ sPAALookupTable[paaLookupTableIdx].mPAACertificate }, out_dac_buffer);
+}
+
+class ExampleDACVerifier : public DeviceAttestationVerifier
+{
+public:
+    AttestationVerificationResult
+    VerifyAttestationInformation(const ByteSpan & attestation_info_buffer, const ByteSpan & attestation_challenge_buffer,
+                                 const ByteSpan & attestation_signature_buffer, const ByteSpan & pai_cert_der_buffer,
+                                 const ByteSpan & dac_cert_der_buffer, const ByteSpan & attestation_nonce) override;
+};
+
+AttestationVerificationResult ExampleDACVerifier::VerifyAttestationInformation(const ByteSpan & attestation_info_buffer,
+                                                                               const ByteSpan & attestation_challenge_buffer,
+                                                                               const ByteSpan & attestation_signature_buffer,
+                                                                               const ByteSpan & pai_cert_der_buffer,
+                                                                               const ByteSpan & dac_cert_der_buffer,
+                                                                               const ByteSpan & attestation_nonce)
+{
+    // match DAC and PAI VIDs
+    if (!pai_cert_der_buffer.empty())
+    {
+        VendorId paiVid;
+        VendorId dacVid;
+
+        CHIP_ERROR error = ExtractVIDFromX509Cert(pai_cert_der_buffer, paiVid);
+        VerifyOrReturnError(error == CHIP_NO_ERROR || error == CHIP_ERROR_KEY_NOT_FOUND,
+                            AttestationVerificationResult::kPaiFormatInvalid);
+
+        if (error == CHIP_NO_ERROR)
+        {
+            VerifyOrReturnError(ExtractVIDFromX509Cert(dac_cert_der_buffer, dacVid) == CHIP_NO_ERROR,
+                                AttestationVerificationResult::kDacFormatInvalid);
+
+            VerifyOrReturnError(paiVid == dacVid, AttestationVerificationResult::kDacVendorIdMismatch);
+        }
+    }
+
+    P256PublicKey remoteManufacturerPubkey;
+    VerifyOrReturnError(ExtractPubkeyFromX509Cert(dac_cert_der_buffer, remoteManufacturerPubkey) == CHIP_NO_ERROR,
+                        AttestationVerificationResult::kDacFormatInvalid);
+
+    // Step d
```

Comment:
Suggest renaming to `// Validate overall attestation signature on attestation informatuon`. Numbered steps can go stale and are not very descriptive

Reply:
agreed, done.